src/App/Pages/Send/SendAddEditPage.xaml.cs

```diff
@@ -113,7 +114,7 @@ protected override void OnDisappearing()
 
         private void TextType_Clicked(object sender, EventArgs eventArgs)
         {
-            _vm.TypeChanged(SendType.Text);
+            _vm.TypeChangedAsync(SendType.Text);
```

Comment:
does this method need to be made `async void` with an `await _vm.TypeChangedAsync(SendType.Text);` call in there? OR, do you need to add `.GetAwaiter().GetResult()` to this async call to make it syncronous?

Reply:
It totally does.